Server/src/main/java/org/openas2/partner/XMLPartnershipFactory.java

```diff
@@ -270,10 +319,9 @@ public void storePartnership() throws OpenAS2Exception {
                 break;
             }
             l++;
+            logger.debug("Backing up " + fn + " to " + f.getName());
         }
 
-        logger.info("Backing up " + fn + " to " + f.getName());
-
```

Comment:
This change to the logger staement being within the loop is not valid. It will skip logging the actual file name it is backing up to.